addons/sale/controllers/variant.py

```diff
@@ -1,6 +1,7 @@
-# -*- coding: utf-8 -*-
```

Comment:
Unrelated diff, leave the line ;)

Reply:
Sure thing. It's a Python2-ism, thought I would get it since I was touching the file.